src/pages/frSetup.page

```diff
@@ -131,5 +204,119 @@
                 </apex:commandLink>
             </apex:pageBlockSection>
         </apex:pageBlock>
+        <apex:pageBlock title="Donor --> Person Account" rendered="{!personAccountsEnabled == true}">
```

Comment:
I would think this would be better separately as combining them would mean including custom labels or adding more methods in the controller to control what text to show making it more complex overall.